### PR TITLE
Add link to doxygen generated docs hosted on wiki.ddnet.tw

### PR DESCRIPTION
**Pull request by edg-l**
I plan on documenting more stuff (in a manner that @heinrich5991 is fine with it) so i think it would be great to mention it on the readme.

The docs are regenerated every 30 minutes using latest master.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ Development discussions happen on #ddnet on Quakenet ([Webchat](http://webchat.q
 
 You can get binary releases on the [DDNet website](https://ddnet.tw/downloads/), find it on [Steam](https://store.steampowered.com/app/412220/DDraceNetwork/) or [install from repository](#installation-from-repository).
 
+You can view the doxygen generated docs at https://wiki.ddnet.tw/docs/
```
Comment (def-) [inline README.md:9]: Something like `[Source Code Documentation](https://ddnet.tw/docs) is available.`

**Comment by def-** [comment]
Btw, why put the documentation under wiki? Is the doxygen script hard to run? Otherwise we could also put it straight on ddnet.tw/docs 

On the docs site itself:

> DDraceNetwork Docs Documentation

Remove the "Docs"

**Comment by edg-l** [comment]
My doxyfile is different from the one in this repo, and it uses a custom theme + has clang assisted parsing enabled. I can send it to you if you want to host it

File: README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ Development discussions happen on #ddnet on Quakenet ([Webchat](http://webchat.q
 
 You can get binary releases on the [DDNet website](https://ddnet.tw/downloads/), find it on [Steam](https://store.steampowered.com/app/412220/DDraceNetwork/) or [install from repository](#installation-from-repository).
 
-You can view the doxygen generated docs at https://wiki.ddnet.tw/docs/
+[Source Code Documentation](https://ddnet.tw/docs) is available.
```
Comment (def-) [inline README.md:9]: probably wiki for now if I don't host it yet.

Comment (edg-l) [reply README.md]: ah ye didn't notice

**Pull request merged**